S1_NRB/nrb.py

```diff
@@ -299,6 +300,8 @@ def format(config, product_type, scenes, datadir, outdir, tile, extent, epsg, wb
                 create_vrt(src=[item, gs_path], dst=sigma0_rtc_lin, fun='mul',
                            relpaths=True, options=vrt_options, overviews=overviews,
                            overview_resampling=ovr_resampling)
```

Comment:
This VRT-layer is added to `datasets_ard` because you need it in line 346, right? It will now end up as a duplicate entry in `ard_assets` (line 379), which is forwarded to the metadata parsing functions. I might be wrong, but I think it will then lead to a duplicate entry in the product XML file. I guess we could just add all of the `.vrt`-layers in this for-loop and skip the `finder(ard_dir, ['.vrt$'], regex=True, recursive=True)`-part of line 379.

Reply:
Sorry, seems like it doesn't show the correct line I was commenting. I'm referring to line 304 here: 
```python
datasets_ard[key] = sigma0_rtc_lin
```

Reply:
done: [eff08e7](https://github.com/SAR-ARD/S1_NRB/pull/138/commits/eff08e7a19a260a8239be7773472194db14913b4)

Reply:
Looks good, thank you!